js/deer-config.js

```diff
@@ -599,7 +609,7 @@ export default {
                         }
                         for(const prop in query){
                             if(li.hasAttribute(`data-${prop}`)){
-                                const action = li.getAttribute(`data-${prop}`).includes(query[prop]) ? "remove" : "add"
+                                const action = li.getAttribute(`data-${prop}`).toLowerCase().includes(query[prop]) ? "remove" : "add";
```

Comment:
You can make the code a lot simpler if you just make `query[prop]` lowercase here.

Reply:
remove the `;`

Reply:
The solution is in just this line -- there should be no additional lines of code necessary.  You still need to remove the semicolon.